### ISSUE TITLE
Epoch length?

### ISSUE THREAD

**Issue by raunak95** [yhenon/keras-frcnn#49]
What is the parameter epoch_length? How is it different from num_epochs?

**Comment by yhenon** [comment]
`epoch_length` is how many images are trained on per epoch. The default value is 1000. At the end of the epoch, weights will be saved if the loss improved.
`num_epochs` is how many epochs to do.
